src/ctap/apdu.rs

```diff
@@ -58,8 +59,8 @@ pub struct ApduHeader {
     p2: u8,
 }
 
-impl From<&[u8]> for ApduHeader {
-    fn from(header: &[u8]) -> Self {
+impl From<&[u8; 4]> for ApduHeader {
+    fn from(header: &[u8; 4]) -> Self {
```

Comment:
This `4` should be `APDU_HEADER_LEN` here and below.

Reply:
Addressed in b9ffe7e4ce0664a0ebe4e5088efbbaa16a75e1a0